Visual Studio/Realm.vcxproj

```diff
@@ -2599,4 +2599,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
+</Project>
```

Comment:
It's github that warns that the file is lacking a terminating empty line. Ignore it